LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2017 Tony Morris, Runar Bjarnason, Tom Adams, Kristian Domagala, Brad Clow, Ricky Clarkson, Paul Chiusano, Trygve Laugstøl, Nick Partridge, Jason Zaugg, Julien Truffaut, Aloïs Cochard, Kenji Yoshida, Vincent Marquez, JB Giraudeau
+Copyright (c) 2009-2018 Tony Morris, Runar Bjarnason, Tom Adams, Kristian Domagala, Brad Clow, Ricky Clarkson, Paul Chiusano, Trygve Laugstøl, Nick Partridge, Jason Zaugg, Julien Truffaut, Aloïs Cochard, Kenji Yoshida, Vincent Marquez, JB Giraudeau
```

Comment:
Not to start anything but are these the people that we consider (and want to be named as) Scalaz 8 copyright holders?

Reply:
I have no opinion. There was discussion on #1783 about this re: the 7.x branches, but it went nowhere concrete.

Reply:
For what it's worth, a quick analysis of `git blame` shows the following people as contributors to the `series/8.0.x` branch:

```bash
hhoughton:scalaz hhoughton$ git rev-list --format=%aN HEAD | grep -v '^commit' | sort -u | sed -e 's/^/- /'
```

- Alexander Konovalov
- Alois Cochard
- Brian McKenna
- Colin Woodbury
- Dale Wijnand
- Edmund Noble
- Emily Pillmore
- Filippo Mariotti
- Gaston Tonietti
- Harrison Houghton
- Igor Mielientiev
- Jean-Baptiste Giraudeau
- John A. De Goes
- Julien Truffaut
- Kamil Kloch
- Nic Flores
- Paweł Sączawa
- Stephen Compall
- Sukant Hajra
- Tim Steinbach
- Tomas Mikula
- Vincent
- Vincent Marquez
- kenji yoshida
- vincent
- wi101
- xuwei-k